packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -76,6 +90,62 @@ class HorizontalOverflow extends Foundation<HorizontalOverflowProps,  React.AllH
         this.setState({
             itemsHeight
         });
+
+        if (canUseDOM() && this.horizontalOverflowItemsRef.current) {
+            this.horizontalOverflowItemsRef.current.addEventListener("scroll", this.requestFrame);
+        }
+    }
+
+    /**
+     * React life-cycle method
+     */
+    public componentWillUnmount(): void {
+        if (canUseDOM() && this.horizontalOverflowItemsRef.current) {
+            this.horizontalOverflowItemsRef.current.removeEventListener("scroll", this.requestFrame);
+        }
+    }
+
+    /**
+     * Callback on scroll change
+     */
+    public onScrollChange = (): IScrollObject => {
+        let scrollObject: IScrollObject;
+        const distanceRemaining: number =
+            this.horizontalOverflowItemsRef.current.scrollWidth - this.horizontalOverflowItemsRef.current.scrollLeft;
+
+        if (this.props.onScrollChange) {
+            this.props.onScrollChange();
+        }
+
+        if (this.horizontalOverflowItemsRef.current.scrollLeft === 0) {
+            this.getLTR() === Direction.ltr ?
+            scrollObject = {
```

Comment:
I don't think you want to assign different values here based on rtl - I would expect "start" to be the left edge in ltr and the right edge in rtl. This implementation would require me to check for "start" in ltr and "end" in rtl when trying to determine if I'm at the beginning of the overflow.

Reply:
Oops - was reading this wrong. Please disregard.